test/e2e/scheduler_predicates.go

```diff
@@ -136,30 +136,17 @@ func cleanupPods(c *client.Client, ns string) {
 }
 
 var _ = Describe("SchedulerPredicates", func() {
+	framework := Framework{BaseName: "sched-pred"}
```

Comment:
The current version of the test (before this PR) does not have a timeout on namespace deletion.  I'd prefer to add the timeout in a separate PR, if it's necessary.  Besides that I don't currently know what a good value for that timeout would be, and don't want to shove an arbitrary one in for no good reason.  OK @gmarek ?